cmd/upgrade/operatorroles/cmd.go

```diff
@@ -185,6 +188,9 @@ func run(cmd *cobra.Command, argv []string) {
 		if reporter.IsTerminal() {
 			reporter.Infof("All policy files saved to the current directory")
 			reporter.Infof("Run the following commands to upgrade the operator IAM policies:\n")
+			if isAccountRoleUpgradeNeed && isProgrammaticallyCalled {
+				reporter.Warnf("Operator roles MUST only be upgraded after Account Roles upgrade has completed.\n")
```

Comment:
Nice catch. We are displaying the warning. What happens if they don't follow.  I think we will be creating two versions of the operator role policies. Same one so not an issue may be.

Reply:
If they don't follow, then there's not much we can do. I'd say leave this warning as is.